superset/assets/javascripts/SqlLab/components/TabbedSqlEditors.jsx

```diff
@@ -6,7 +6,7 @@ import * as Actions from '../actions';
 import SqlEditor from './SqlEditor';
 import CopyQueryTabUrl from './CopyQueryTabUrl';
 import { areArraysShallowEqual } from '../../reduxUtils';
-import { getParamFromQuery } from '../../../utils/common';
+import URI from 'urijs';
```

Comment:
what is the advantage of `urijs` over using `getParamFromQuery`? it does look a little cleaner, but i'm thinking it might not be worth the addition depending on the package size.
how big of a lib is `urijs`?